src/t2iapi/device/device_requests.proto

```diff
@@ -42,4 +42,11 @@ Request a change of the MDIB SequenceId.
 message ChangeMdibSequenceIdRequest {
     string descriptive_name = 1;  // descriptive name as a string which describes the trigger
                                   // for a change of the MDIB SequenceId
+}
+
+/*
+Request a filtered List of removable Descriptors.
```

Comment:
unnecessary capitalization _list_ _descriptors_

Proposal: Request a list of removable descriptors of the given type.

Reply:
Proposal accepted.